### PR TITLE
Added toast alerts to login/register pages, resolves #147

### LINKED ISSUE #147: Frontend - Backend - Use Toast component to display errors
Currently errors on login and registration are shown using `alert` as a placeholder, these can all be replaced with the new `Toast` component.

### PR DESCRIPTION
**Pull request by danielcj2**
Added toast alerts to login/register pages and move `ToastContainer` to top of the page so the alerts can be displayed everywhere in the app.

### CODE REVIEW

File: Client/src/Pages/Register/index.jsx [hunk 1]
```diff
@@ -91,16 +92,26 @@ const Register = () => {
       }
     } catch (error) {
       if (error.name === "ValidationError") {
-        // TODO Handle validation errors
-        console.log(error);
-        alert(error);
+        // validation errors
+        createToast({
+          variant: "info",
+          body: error.details[0].message,
```
Comment (ajhollid) [inline Client/src/Pages/Register/index.jsx:98]: We should do a check here before trying to access items from the `details` array, it's not guaranteed to be populated

**Review by ajhollid — CHANGES_REQUESTED** [review]
Looks good, we should do a check before accessing details array though for safety's sake

**Review by ajhollid — APPROVED** [review]

File: Client/src/Pages/Login/index.jsx [hunk 2]
```diff
@@ -78,7 +78,10 @@ const Login = () => {
         // validation errors
         createToast({
           variant: "info",
-          body: error.details[0].message,
+          body:
+            error && error.details && error.details.length > 0
+              ? error.details[0].message
+              : "Error validating data.",
```
Comment (danielcj2) [inline Client/src/Pages/Login/index.jsx:84]: oh wait I don't think error.details.length > 0 is relevant here

Comment (ajhollid) [reply Client/src/Pages/Login/index.jsx:84]: woops, speed merged it, go ahead and open a fix PR

Comment (danielcj2) [reply Client/src/Pages/Login/index.jsx:84]: Disregard this, I just had a brain fart 🤣 Length will still be > than 0 when we're accessing the 0 index.

Comment (ajhollid) [reply Client/src/Pages/Login/index.jsx:84]: Good ol off by 1 :stuck_out_tongue:

**Pull request merged**